src/System.Windows.Forms/src/System/Windows/Forms/TreeNodeCollection.cs

```diff
@@ -63,6 +63,11 @@ internal int FixedIndex
                     throw new ArgumentOutOfRangeException(nameof(index), index, string.Format(SR.InvalidArgument, nameof(index), index));
                 }
 
+                if(value.Handle != IntPtr.Zero)
```

Comment:
[nit] incorrect spacing
```suggestion
                if (value.Handle != IntPtr.Zero)
```

Reply:
Thank you, I have solved the spacing and trailing whitespace